src/main/java/com/codegym/shopyy/model/entity/User.java

```diff
@@ -9,39 +15,29 @@
 import java.util.Date;
 
 @Entity
-@Table(name = "users")
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
 public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id")
     private Long id;
 
-    @Column(name = "fullname")
-    private String fullName;
+    private String full_name;
```

Comment:
using camelCase instead of snake_case in entity

Reply:
resolved